index.html

```diff
@@ -1,544 +1,627 @@
-      <div class="is-2 title">2023</div>
-      <p>Please take a llok at our plans</p>
-      <table>
-        <tr>
-          <td>Header</td>
-          <td>Header</td>
-        </tr>
-        <tr>
-          <td>cell</td>
-          <td>cell</td>
-        </tr>
-        <tr>
-          <td>cell</td>
-          <td>cell</td>
-        </tr>
-      </table>
-    </div>
-  </div>
+    </nav>
 
-  <div class="hero has-background-grey-light">
-    <div class="container features">
-      <div class="section-heading has-text-centered">
-        <div class="title is-2">Our services</div>
-        <div class="subtitle is-5">We help our customers to increase value</div>
-      </div>
+    <header class="hero">
+      <div class="hero-head">
+        <div class="navbar">
+          <div class="container">
+            <div class="navbar-brand">
+              <a href="/" class="navbar-item logo-link">
+                <img src="./img/logo.png" alt="Best company logo">
+              </a>
+              <button class="navbar-burger burger" data-target="navbarMenu">
+                <span></span>
+                <span></span>
+                <span></span>
+              </button>
+            </div>
 
-      <figure>
-        <img src="./img/IMG_5067_72DPI-RGB_.jpg">
-        <img src="./img/iStock_73271969_web.jpg">
-        <figcaption>
-          Figure 1: Some attractive pictures to ensure our customers
-        </figcaption>
-      </figure>
+            <nav id="navbarMenu" class="navbar-menu">
+              <ul class="navbar-end">
+                <span class="navbar-item">
+                  <a href="/" class="button is-white is-outlined">
+                    <span class="icon">
+                      <i class="fa fa-home"></i>
+                    </span>
+                    <span>Home</span>
+                  </a>
+                </span>
+                <span class="navbar-item">
+                  <a href="/todo"
+                    class="button is-white is-outlined with-sublist">
+                    <span class="icon">
+                      <i class="fab fa-superpowers"></i>
+                    </span>
+                    <span>Services</span>
+                  </a>
+                  <ul class="sublist visually-hidden">
+                    <li>Design</li>
+                    <li>Consult</li>
+                    <li>Engineer</li>
+                    <li>Operate</li>
+                    <li>Optimize</li>
+                  </ul>
+                </span>
+                <span class="navbar-item">
+                  <a href="/todo" class="button is-white is-outlined">
+                    <span class="icon">
+                      <i class="fa fa-book"></i>
+                    </span>
+                    <span>Success stories</span>
```

Comment:
Focus looks inconsistent in different browsers. You should keep it in mind and refer to design in a real project. This is just for your information.

<img width="764" alt="image" src="https://github.com/yurassic/accessibility-training/assets/28926420/8d9e3300-22e7-4b1a-aea2-2c0e0438c0a8">
<img width="739" alt="image" src="https://github.com/yurassic/accessibility-training/assets/28926420/65dfb118-1117-4f64-a5c7-63a6505aad9f">

 What can be done:
```
a:focus-visible {
  outline: none;
  box-shadow: 0 0 2px 1px #fff, 0 0 3px 3px #000;
}
```